modules/ROOT/pages/worker-monitoring.adoc

```diff
@@ -42,11 +42,17 @@ image::chmulesystemhealthmsg.png[CHMuleSystemHealthMsg]
 
 == How Monitoring Works
 
+*Runtime 3.x.y* 
+
```

Comment:
Every 30 seconds, CloudHub sends an HTTP GET request...

Logs should be lowercase--logs